docs/guides/UpgardeGuide-v5/UpgradePart2.md

```diff
@@ -0,0 +1,1012 @@
+  })
+
+  describe('popup', () => {
+    before(() => {
+      browser.url('/#/raw/tests/example');
+      browser.click('#popupButton');
+      browser.waitForExists('#popup');
+    });
+    
+    Terra.it.validatesElement();
+
+    it('click popup content', () => {
+      browser.click('#popupContent')
+      Terra.validates.accessibility();
+      Terra.validates.screenshot();
+    });
+  });
+
+  describe('table', () => {
+    before(() => browser.url('/#/raw/tests/example'));
+    
+    describe('table default')
+      Terra.it.isAccessible();
+      Terra.it.matchesScreenshot();
+    });
+
+    describe('table with first item selected', () => {
+      it('clicks row 1', () => {
+-       browser.click('#row1');
++       browser.leftClick('#row1');
+      });
+
+      Terra.it.validatesElement('row 1 selected');
+    });
+
+    describe('table with second item selected', () => {
+-     it('clicks row 2', () => {
+-       browser.click('#row2');
++     it('tabs and selects row 2', () => {
++       browser.keys(['Tab', 'Enter']);
+      });
+
+      Terra.it.isAccessible();
+      Terra.it.matchesScreenshot('row 2 selected');
+    });
+  });
+});
+```
+</details>
+
+### Now, Lets Slim Down Our Spec
+With the new test helpers, we could actually slim down this spec some more if we really wanted to. If you have interest in doing this, here is what we could now write:
+
+<details>
+<summary>The final, updated and slimmed down spec: </summary>
+
+```js
+Terra.describeViewports('Example', ['tiny', 'huge'], () => {
+  describe('input', () => {
+    it('validates input', () => {
+      browser.url('/#/raw/tests/example');
+      // Removes the blinking cursor to prevent screenshot mismatches.
+      browser.execute('document.querySelector('\'input\').style.caretColor = "transparent";');
+      Terra.validates.element();
+    });
+
+    it('enters text', () => {
+      browser.keys('Tab');
+      browser.addValue('input', 'stuff');
+      Terra.validates.element('with input');
+    });
+  });
+
+  describe('popup', () => {
+    it('validates popup', () => {
+      browser.url('/#/raw/tests/example');
+      browser.click('#popupButton');
+      browser.waitForExists('#popup');
+      Terra.validates.element();
+    });
+    
+    it('click popup content', () => {
+      browser.click('#popupContent')
+      Terra.validates.element();
+    });
+  });
+
+  describe('table', () => {
+    it('validates default table render')
+      browser.url('/#/raw/tests/example')
+      Terra.validates.element();
+    });
+
+    it('clicks row 1 to selected the first item', () => {
+      browser.leftClick('#row1');
+      Terra.validates.element('row 1 selected');
+    });
+
+    it('tabs and selects row 2 to selected the second item', () => {
+      browser.keys(['Tab', 'Enter']);
+      Terra.validates.element('row 2 selected');
+    });
+  });
+});
+```
+</details>
+
+### The Results
+Then, after slimming down the spec, we end up w:
+- 2 viewport resizes
```

Comment:
impressive! want to put the old test results side-by-side with these for comparison?